src/Adapter/PDF/DeliverySlipPdfGenerator.php

```diff
@@ -58,22 +59,24 @@ public function __construct(TranslatorInterface $translator)
     /**
      * {@inheritdoc}
      */
-    public function generatePDF(array $orderId)
+    public function generatePDF(array $orderIds)
     {
-        if (count($orderId) !== 1) {
-            throw new CoreException(sprintf('"%s" supports generating delivery slip for single order only.', get_class($this)));
+        if (!$orderIds) {
+            throw new MissingDataException('Missing data required to generate PDF');
```

Comment:
Detailled error messages are always helpful :)
```suggestion
            throw new MissingDataException('No order ID provided to generate delivery slip PDF');
```